test/testsuites/proxytests.go

```diff
@@ -0,0 +1,67 @@
+package testsuites
+
+import (
+	"context"
+	"testing"
+
+	apiconfigv1 "github.com/openshift/api/config/v1"
+	"github.com/operator-framework/operator-marketplace/test/helpers"
+	"github.com/operator-framework/operator-sdk/pkg/test"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	apps "k8s.io/api/apps/v1"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+// ProxyTests is a test suite that ensure that the watches for proxy resources
+// are firing correctly and the deployment resources are updated as a result.
+func ProxyTests(t *testing.T) {
+	t.Run("no-proxy", testSetNoProxyVariable)
+}
+
+// testSetNoProxyVariable creates an OperatorSource and then set the cluster proxy.Spec.NoProxy
```

Comment:
Where are you creating the `OperatorSource` in this test case?

Reply:
This test is ran as part of the `OperatorSource` TestGroup, which creates an `OperatorSource` named `test-operators` prior to running this test.

Reply:
I know. You should remove the statement that you are creating the `OperatorSource` in this function.

Reply:
Ahh